.gitignore

```diff
@@ -35,4 +35,8 @@ searchdb/*
 nbproject/*
 
 plugin/bbb/config.vm.php
+
+# Extrafields files
+main/upload/extrafields/*
```

Comment:
When moving it to main/upload/users/, main/upload/courses/ and main/upload/sessions, don't forget to update the .gitignore rule here